.github/workflows/ci.yaml

```diff
@@ -234,8 +283,18 @@ jobs:
                     cat ${{ github.workspace }}/.github/workflows/ci.bazel5.bazelrc >> ${{ github.workspace }}/.github/workflows/ci.bazelrc
                   fi
 
-            - name: Write EngFlow credentials
-              # Writes EngFlow credential files for RBE configurations
+            - name: Configure Bazel version (Windows)
+              if: matrix.os == 'windows-latest'
+              working-directory: ${{ matrix.folder }}
+              # Overwrite the .bazelversion instead of using USE_BAZEL_VERSION so that Bazelisk
+              # still bootstraps Aspect CLI from configuration in .bazeliskrc. Aspect CLI will
+              # then use .bazelversion to determine which Bazel version to use.
+              run: |
+                  echo "${{ matrix.bazelversion }}" > .bazelversion
+                  $BAZEL_MAJOR_VERSION = "${{ matrix.bazelversion }}".substring(0, 1)
```

Comment:
Where is `BAZEL_MAJOR_VERSION` used? That's not used with non-windows?

Reply:
Nice catch. That is a copy-paste left-over from https://github.com/aspect-build/bazel-lib/blob/794df714d7efbf5f2b986470428bea311f4fd772/.github/workflows/ci.yaml#L168 where it is used.